modules/gawk/5.3.2/source.json

```diff
@@ -0,0 +1,12 @@
+{
+    "url": "https://ftp.gnu.org/gnu/gawk/gawk-5.3.2.tar.xz",
```

Comment:
This site could be slow and unresponsive. Maybe we could use an auto-chosen mirror as described by https://www.gnu.org/prep/ftp.en.html?